src/main/java/ikura/task/Event.java

```diff
@@ -0,0 +1,76 @@
+// Event.java
+// Copyright (c) 2020, zhiayang, Apache License 2.0.
+
+package ikura.task;
+
+import java.util.Scanner;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import ikura.util.InvalidInputException;
+
+/**
+ * A class representing a Event task. It has a description (name), and a date.
+ */
+public class Event extends Task {
+
+	private final LocalDate eventDate;
+
+    /**
+     * Constructs a new Event task with the given description and date.
+     *
+     * @param name     the Event's description.
+     * @param deadline the date of this event.
+     */
+    public Event(String name, LocalDate date) {
+        super(name);
+		this.eventDate = date;
```

Comment:
Some spacing issues here

Reply:
ah yes, my editor is configured to tabs by default, i must've missed this one. thanks!